### PR TITLE
NTR liver resident NK cells

### PR DESCRIPTION
**Pull request by JABelfiore**
new term added and refs

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

**Review by dosumis — CHANGES_REQUESTED** [review]
The markers are for humans, so we need to make that clear in the text.

(we will soon be able to formally record these in [CellMark](https://github.com/Cellular-Semantics/CellMark) 

If we define this term with an equivalentClass axiom, hepatic pit cells will be a subclass, which I think is what the author of the ticket is suggesting. 

Hepatic pit cells are located in the hepatic sinusoid.  Is this true of the general class?

Hepatic pit cell terms should have some [taxon constraints](https://oboacademy.github.io/obook/explanation/taxon-constraints-explainer/):

present_in_taxon: Rattus rattus (Or broaden to rodents?)
never_in_taxon: Homo sapiens

**Comment by dosumis** [comment]
#gogoedidiff

**Comment by JABelfiore** [comment]
#gogoedidiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32258,6 +32260,18 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4047001 (liver-resident natural killer cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26330348") Annotation(oboInOwl:hasDbXref "PMID:26639736") Annotation(oboInOwl:hasDbXref "PMID:27798170") Annotation(oboInOwl:hasDbXref "PMID:32351704") Annotation(oboInOwl:hasDbXref "PMID:34230995") Annotation(oboInOwl:hasDbXref "PMID:35865550") obo:IAO_0000115 obo:CL_4047001 "A natural killer cell resident to the liver, in humans this cell type is distinguished from circulating natural killer cells by CD49a or CD69 gene expression. Liver-resident natural killer cells have also been shown to express CCR5, EOMES, KLRB1, GZMK, and CXCR6 in humans.")
+AnnotationAssertion(terms:contributor obo:CL_4047001 <https://orcid.org/0009-0005-7919-4905>)
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:32266]: Add as **related_synonym:**
liver-specific natural killer cell 
PMID: 26330348

the definition states  `Liver-resident natural killer cells` but the cell label is `liver resident natural killer cell` remove `-` and add `liver-resident natural killer cell` as a related synonym

optional: consider adding a `comment`  section for the role of liver resident natural killer cells during inflammation. You can consider looking at this reference PMID:34230995

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32258,6 +32260,18 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
...
+AnnotationAssertion(terms:contributor obo:CL_4047001 "https://orcid.org/0000-0001-8134-3037")
+AnnotationAssertion(terms:date obo:CL_4047001 "2024-07-29T13:43:11Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26639736") oboInOwl:hasExactSynonym obo:CL_4047001 "liver NK cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35726345") oboInOwl:hasExactSynonym obo:CL_4047001 "lr-NK")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32351704") Annotation(oboInOwl:hasDbXref "PMID:34230995") oboInOwl:hasExactSynonym obo:CL_4047001 "lrNK")
+AnnotationAssertion(rdfs:label obo:CL_4047001 "liver-resident natural killer cell")
+EquivalentClasses(obo:CL_4047001 ObjectIntersectionOf(obo:CL_0000623 ObjectSomeValuesFrom(obo:RO_0001025 obo:UBERON_0002107)))
+
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:32274]: these might be other synonyms for liver resident natural killer cells:

- hepatic natural killer cells
- hepatic NK
- he-NK
PMID: [31114585](https://pubmed.ncbi.nlm.nih.gov/31114585)

- intrahepatic NK
PMID: [33828559](https://pubmed.ncbi.nlm.nih.gov/33828559)

**Review by AvolaAmg — CHANGES_REQUESTED** [review]
Please see in line comments

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:32274]: Thanks Alida, added!

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:32266]: added synonym and com ent. Did not remove '- ' in description as cell label is 'liver-resident natural killer cell', but did add 'liver resident natural killer cell' as synonym

**Comment by dosumis** [comment]
I think this is the last remaining term lost to clashing ID issue.  I'll try to recover.

**Comment by dosumis** [comment]
@gouttegd - great to see this taxon constraints running.  Trying to work out reason for this violation.

I don't see where this is coming from:

image

hepatic pit cell has:

image
natural killer cell has
image

As I understand it 'present in taxon' is very weak.  It just means that some cells of this type exist in the taxon, not all and so doesn't clash with the "never in taxon" on the more specific cell type.

**Review by AvolaAmg — DISMISSED** [review]

**Comment by dosumis** [comment]
Thanks for tracing - I think we just need an import statement to ensure that the component axioms are visible to editors in Protege.

**Comment by gouttegd** [comment]
The `hra_subset` component _is_ already imported in `cl-edit.owl` -- its axioms _are_ visible when you open `cl-edit.owl` in Protégé.

In fact this is probably how the component got changed in commit https://github.com/obophenotype/cell-ontology/pull/2472/commits/735125198519b71827d27daeb33ff7442a8ce5ff. I assume the editor saw the `present_in_taxon 'Homo sapiens'` annotation in Protégé and simply changed it from there (possibly without realising it came from a component rather than the -edit file itself).

**Review by dosumis — APPROVED** [review]
pushing through

**Pull request merged**